### PR TITLE
Add 2022Q4 entries from clusteradm

### PR DESCRIPTION
**Pull request by dbaio**

### CODE REVIEW

**Review by pauamma — COMMENTED** [review]
All minor changes, fixable while/after merging if you agree, to save a round of edits.

File: 2022q4/clusteradm.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+In this quarter, the team has worked on the following:
+
+* Regular cluster-wide software upgrades +
+Two full upgrades to fix and prevent some impacting issues (FreeBSD-EN-22:25.tcp and FreeBSD-EN-22:28.heimdal).
+* Regular support for FreeBSD.org user accounts
+* Regular disk and parts support (and replacement) for all physical hosts and mirrors.
+* Site audit at our primary site
+** Inventory of spares and other miscellanea occupying space in our cabinets.
+** Inventory of PDUs/power outlet usage and identify faulty PSUs.
```
[the remark this answers is not in the document]
Comment (dbaio) [reply 2022q4/clusteradm.adoc]: s/identify/identifying/
done

**Comment by dbaio** [comment]
Thanks. I've pushed the wording fixes.

About the other suggestions (in the structure), look at the document after rendering ([here](https://people.freebsd.org/~dbaio/clusteradm.html)).
The list items are here as fake titles. I added a `+` to keep the next paragraph indented.

**Review by pauamma — APPROVED** [review]
Fair enough.

**Pull request merged**